internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -345,6 +345,43 @@ func (provisioner *KopsProvisioner) ProvisionCluster(cluster *model.Cluster, aws
 			DeployNamespace: "kube-system",
 		},
 	}
+
+	_, err = k8sClient.Clientset.CoreV1().Namespaces().Get(ctx, "pgbouncer", metav1.GetOptions{})
+	if k8sErrors.IsNotFound(err) {
```

Comment:
Is there a reason you didn't want to use `CreateOrUpdateNamespace` for this? I has the check in it to see if the namespace exists.

Reply:
A follow up question I have is why is there some pgbouncer provisioning logic hardcoded here? I see that you were able to add pgbouncer as a utility which is awesome, but I am curious why the following change isn't done within that utility code instead?

Reply:
Thank you, @gabrieljackson, for both of your suggestions you were absolutely right, I have updated the code, so the pgbouncer manifests are created alongside with utility and I have used the `CreateOrUpdateNamespace`